opencti-platform/opencti-graphql/tests/01-unit/domain/csv-bundler-test.ts

```diff
@@ -7,65 +7,70 @@ import {
   indicatorsWithExternalReferencesCsvMapper
 } from '../../data/csv-bundler/external-references-constants';
 import type { CsvMapperParsed } from '../../../src/modules/internal/csvMapper/csvMapper-types';
+import { indicatorsWithLabelsCsvContent, indicatorsWithLabelsCsvMapper } from '../../data/csv-bundler/labels-constants';
+import {
+  indicatorsWithKillChainPhasesCsvContent,
+  indicatorsWithKillChainPhasesCsvMapper
+} from '../../data/csv-bundler/kill-chains-constants';
 
 describe('CSV bundler', () => {
   describe('Embedded properties', () => {
-    it('Should list external references', async() => {
+    it('Should list external references', async () => {
       const indicatorsWithExternalReferencesExpectedBundle: StixBundle = {
```

Comment:
I think you can put the bundles in a file under data too, to avoid making this file too big.